lntest/btcd.go

```diff
@@ -88,7 +88,7 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		// make sure they stay connected if it happens.
 		"--nobanning",
 	}
-	chainBackend, err := rpctest.New(netParams, nil, args)
+	chainBackend, err := rpctest.New(netParams, nil, args, GetBtcdBinary())
```

Comment:
what benefit does this give us? Mostly for local testing?

Reply:
It avoids the test harness compiling `btcd` for every run by invoking `go build`. That way we also have more certainty that the actually built version of `btcd` is what we define in `go.mod`. Also we don't overwrite any pre-installed `btcd` on the system anymore.